src/components/Products/ProductImage.js

```diff
@@ -3,21 +3,35 @@ import React, { Component } from 'react';
 class ProductImage extends Component {
 
   render() {
-    var placeholder = 'https://placeholdit.imgix.net/~text?txtsize=69&txt=824%C3%971050&w=824&h=1050';
-
-    try {
-      var fileId = this.props.product.relationships.files.data[0].id;
-      var file = this.props.products.included.files.find(function (el) {
+    
+    let file;
+    let fileId;
+    let placeholder = 'https://placeholdit.imgix.net/~text?txtsize=69&txt=824%C3%971050&w=824&h=1050';
+    
+    var isThereAMainImage = (product) => {
+      fileId = this.props.product.relationships.main_image.data.id;
+      
+      file = this.props.products.included.main_images.find(function (el) {
         return fileId === el.id
       })
-      
-      return <img alt="" src={file.link.href} style={{"background": this.props.background}}/> || <img alt="" src={placeholder}/>
 
+      return <img alt={this.props.product.name + '-' + this.props.product.description} src={file.link.href} style={{"background": this.props.background}}/> || <img alt="placeholder" src={placeholder}/>
+    };
+    
+    var isThereAFile = (product) => {
+      fileId = this.props.product.relationships.files.data[0].id;
+      file = this.props.products.included.files.find(function (el) {
+        return fileId === el.id
+      })      
+        return <img alt={'product name:' + this.props.product.name + ', product description:' + this.props.product.description} src={file.link.href} style={{"background": this.props.background}}/> || <img alt="placeholder" src={placeholder}/>
```

Comment:
Because it reads really strangely. It is clear to you that the name is one thing and that you go into the description (as the description is really long).